sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala

```diff
@@ -739,6 +739,123 @@ class ColumnarBatchSuite extends SparkFunSuite {
     }}
   }
 
+  test("Nest Array in Array.") {
```

Comment:
Sure. My previous comment was a little bit wrong.

What I want to say is the following. It would be good to prepare the following two cases for array test.
In the case of `new ArrayType(new ArrayType(IntegerType, false), true)`, all of the elements in each leaf array must not have `null` (i.e. this test case).
In the case of `new ArrayType(new ArrayType(IntegerType, true), true)`, any element in each leaf array may have `null` (e.g. `[[0]], [[1, 2], []], [[], [3, null, 5]]`).